guides/common/modules/proc_configuring-project-for-performance.adoc

```diff
@@ -3,3 +3,5 @@
 
 {Project} comes with a number of components that communicate with each other.
 You can tune these components independently of each other to achieve the maximum possible performance for your scenario.
+
+We have not seen significant performance difference between {Project} installed on Red Hat Enterprise Linux 7 and Red Hat Enterprise Linux 8.
```

Comment:
Please use the correct macros
```suggestion
We have not seen significant performance difference between {Project} installed on {EL} 7 and {EL} 8.
```

I also think the docs avoid `we` since it's ambiguous. @maximiliankolb this isn't written down in `CONTRIBUTING.md`. Yesterday I was talking about this: documenting our documentation styles would be a good thing to have. Is this something's done?